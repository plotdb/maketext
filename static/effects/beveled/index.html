<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 500 150"><defs><filter id="beveled" filterUnits="objectBoundingBox" primitiveUnits="userSpaceOnUse"><feGaussianBlur stdDeviation="5" in="SourceGraphic" result="BLUR"><feSpecularLighting surfaceScale="6" specularConstant="1" specularExponent="30" lighting-color="#white" in="BLUR" result="SPECULAR"><fePointLight x="40" y="-30" z="200"></fePointLight></feSpecularLighting><feComposite operator="in" in="SPECULAR" in2="SourceGraphic" result="COMPOSITE"></feComposite><feMerge><feMergeNode in="SourceGraphic"></feMergeNode><feMergeNode in="COMPOSITE"></feMergeNode></feMerge></feGaussianBlur></filter></defs><text x="250" y="75" filter="url(#beveled)" fill="#eee">Hello World!</text></svg>