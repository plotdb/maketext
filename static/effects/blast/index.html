<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 500 150"><filter id="blast" filterUnits="userSpaceOnUse"><feFlood flood-color="red" result="COLOR-white"></feFlood><feFlood flood-color="green" result="COLOR-black"></feFlood><feFlood flood-color="yellow" result="COLOR-third"></feFlood><feFlood flood-color="rgba(0,0,0,0)" flood-opacity="0" result="TRANSPARENT"></feFlood><feMorphology operator="dilate" radius="4" in="SourceGraphic" result="FAT-OUTLINE_10"></feMorphology><feConvolveMatrix in="FAT-OUTLINE_10" order="6,5" divisor="1" kernelMatrix="0 0 1 1 0 0 0 1 1 1 1 0 1 1 1 1 1 1 0 1 1 1 1 0 0 0 1 1 0 0 " result="FAT-OUTLINE_20"></feConvolveMatrix><feComposite operator="in" in="COLOR-black" in2="FAT-OUTLINE_20" result="FAT-OUTLINE_25"></feComposite><feOffset dx="4" dy="0" in="FAT-OUTLINE_25" result="FAT-OUTLINE_30"></feOffset><feImage href="data:image/svg+xml;charset=utf-8,%3Csvg%20version%3D%221.1%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%20width%3D%22100px%22%20height%3D%22200px%22%20%20%3E%0A%09%3Cdefs%3E%0A%09%09%3Cpattern%20id%3D%22pattern%22%20patternUnits%3D%22userSpaceOnUse%22%20width%3D%225px%22%20height%3D%225px%22%20viewBox%3D%220%200%205%205%22%20%3E%0A%09%09%09%3Cpolygon%20points%3D%225%2C0%204.012%2C0%205%2C0.989%20%09%22%2F%3E%0A%09%09%09%3Cpolygon%20points%3D%225%2C5%200%2C0%200%2C0.989%204.01%2C5%20%09%22%2F%3E%0A%09%09%3C%2Fpattern%3E%0A%09%3C%2Fdefs%3E%0A%09%3Crect%20x%3D%220%22%20y%3D%220%22%20width%3D%22100%25%22%20height%3D%22100%25%22%20fill%3D%22url(%23pattern)%22%20%2F%3E%0A%3C%2Fsvg%3E" x="0" y="2" width="100" height="200" result="STRIPED-SHADOW_10"></feImage><feTile in="STRIPED-SHADOW_10" result="STRIPED-SHADOW_20"></feTile><feOffset dx="7" dy="8" in="FAT-OUTLINE_30" result="STRIPED-SHADOW_30"></feOffset><feComposite operator="in" in="STRIPED-SHADOW_20" in2="STRIPED-SHADOW_30" result="STRIPED-SHADOW_40"></feComposite><feComposite operator="in" in="COLOR-black" in2="STRIPED-SHADOW_40" result="STRIPED-SHADOW_50"></feComposite><feImage href="data:image/svg+xml;charset=utf-8,%3Csvg%20version%3D%221.1%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%20width%3D%22100px%22%20height%3D%22200px%22%20%20%3E%0A%09%3Cdefs%3E%0A%09%09%3Cpattern%20id%3D%22pattern%22%20patternUnits%3D%22userSpaceOnUse%22%20width%3D%2210%22%20height%3D%2210%22%3E%0A%0A%09%09%09%3Cpath%20fill%3D%22%23FFFFFF%22%20d%3D%22M0%2C8.239V10h1.761L0%2C8.239z%22%2F%3E%0A%09%09%09%3Cpath%20fill%3D%22%23FFFFFF%22%20d%3D%22M5%2C0l5%2C5l0%2C0V3.238L6.762%2C0H5z%22%2F%3E%0A%09%09%09%3Cpolygon%20fill%3D%22%23FFFFFF%22%20points%3D%220%2C3.239%200%2C5%205%2C10%206.761%2C10%20%22%2F%3E%0A%09%09%09%3Cpolygon%20fill%3D%22%23FFFFFF%22%20points%3D%221.762%2C0%200%2C0%2010%2C10%2010%2C8.238%20%22%2F%3E%0A%09%09%3C%2Fpattern%3E%0A%09%3C%2Fdefs%3E%0A%09%3Crect%20x%3D%220%22%20y%3D%220%22%20width%3D%22100%25%22%20height%3D%22100%25%22%20fill%3D%22url(%23pattern)%22%20%2F%3E%0A%3C%2Fsvg%3E" x="0" y="2" width="100" height="200" result="STRIPED-FILL_10"></feImage><feTile in="STRIPED-FILL_10" result="STRIPED-FILL_20"></feTile><feComposite operator="in" in="STRIPED-FILL_20" in2="SourceAlpha" result="STRIPED-FILL_30"></feComposite><feComposite operator="in" in="COLOR-white" in2="STRIPED-FILL_30" result="STRIPED-FILL_40"></feComposite><feGaussianBlur stdDeviation="4" in="SourceAlpha" result="REFLECTION_10"></feGaussianBlur><feSpecularLighting surfaceScale="7" specularConstant="1.8" specularExponent="30" lighting-color="#white" in="REFLECTION_10" result="REFLECTION_20"><fePointLight x="-100" y="-150" z="250"></fePointLight></feSpecularLighting><feComposite in2="SourceAlpha" operator="in" in="REFLECTION_20" result="REFLECTION_25"></feComposite><feComposite in2="REFLECTION_25" operator="in" in="COLOR-third" result="REFLECTION_30"></feComposite><feMerge result="merge2"><feMergeNode in="STRIPED-SHADOW_50"></feMergeNode><feMergeNode in="FAT-OUTLINE_30"></feMergeNode><feMergeNode in="STRIPED-FILL_40"></feMergeNode><feMergeNode in="REFLECTION_30"></feMergeNode><feMergeNode in="TRANSPARENT"></feMergeNode></feMerge></filter><text x="250" y="75" filter="url(#blast)" fill="#000">Hello World!</text></svg>