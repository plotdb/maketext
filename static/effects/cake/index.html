<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 500 150" style="background:#b55500"><defs><filter id="cake" x="-100%" y="-100%" width="300%" height="300%"><feFlood flood-color="#2d1500" result="flood-shadow"></feFlood><feFlood flood-color="#894619"></feFlood><feComposite operator="in" in2="SourceAlpha"></feComposite><feConvolveMatrix order="7,10" divisor="1" result="conv" kernelMatrix="0 0 0 1 0 0 0 0 0 1 0 1 0 0 0 0 1 0 1 0 0 0 1 0 0 0 1 0 0 1 0 0 0 1 0 0 1 0 0 0 1 0 1 0 0 0 0 0 1 1 0 0 0 0 0 1 0 1 1 0 1 1 0 0 0 0 1 0 0 0 "></feConvolveMatrix><feGaussianBlur stdDeviation="0.5"></feGaussianBlur><feOffset dy="4" result="offset"></feOffset><feComposite operator="in" in="flood-shadow" in2="offset"></feComposite><feGaussianBlur stdDeviation="4"></feGaussianBlur><feComponentTransfer><feFuncA type="linear" slope="0.7" intercept="-0.2"></feFuncA></feComponentTransfer><feOffset dy="4" result="shadow"></feOffset><feGaussianBlur in="SourceGraphic" stdDeviation="0.5"></feGaussianBlur><feOffset dy="0" result="x0"></feOffset><feGaussianBlur in="SourceGraphic" stdDeviation="1"></feGaussianBlur><feComponentTransfer><feFuncA type="linear" slope="0.8" intercept="-.01"></feFuncA></feComponentTransfer><feOffset dy="1" result="x1"></feOffset><feGaussianBlur in="SourceGraphic" stdDeviation="2"></feGaussianBlur><feComponentTransfer><feFuncA type="linear" slope="0.7" intercept="-.01"></feFuncA></feComponentTransfer><feOffset dy="2" result="x2"></feOffset><feGaussianBlur in="SourceGraphic" stdDeviation="3"></feGaussianBlur><feComponentTransfer><feFuncA type="linear" slope="0.6" intercept="-.01"></feFuncA></feComponentTransfer><feOffset dy="3" result="x3"></feOffset><feMerge><feMergeNode in="shadow"></feMergeNode><feMergeNode in="offset"></feMergeNode><feMergeNode in="x3"></feMergeNode><feMergeNode in="x2"></feMergeNode><feMergeNode in="x1"></feMergeNode><feMergeNode in="x0"></feMergeNode></feMerge></filter></defs><text x="250" y="75" filter="url(#cake)" fill="#ffd800">Hello World!</text></svg>