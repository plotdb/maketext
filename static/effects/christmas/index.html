<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 500 150" style="background:#ddd;"><defs><filter id="christmas" x="-100%" y="-100%" width="300%" height="300%"><feGaussianBlur in="SourceAlpha" stdDeviation="3" result="blur"></feGaussianBlur><feSpecularLighting surfaceScale="4" specularConstant="2" specularExponent="5" lighting-color="#fff" in="blur" result="specular"><fePointLight x="0" y="-25000" z="1"></fePointLight></feSpecularLighting><feGaussianBlur in="specular" stdDeviation="3" result="specular2"></feGaussianBlur><feComponentTransfer in="specular2" result="specular3"><feFuncA type="linear" slope="2.5" intercept="-1"></feFuncA></feComponentTransfer><feComponentTransfer in="specular3" result="specular4"><feFuncA type="linear" slope="2" intercept="-1"></feFuncA></feComponentTransfer><feImage x="0" y="0" width="12" height="12" href="data:image/svg+xml,&lt;svg xmlns=&quot;http://www.w3.org/2000/svg&quot; width=&quot;10&quot; height=&quot;10&quot; viewBox=&quot;0 0 10 10&quot;&gt;&lt;rect x=&quot;-1&quot; y=&quot;-1&quot; width=&quot;12&quot; height=&quot;12&quot; fill=&quot;#fff&quot;/&gt;&lt;path d=&quot;M0 0L10 10M-10 0 L0 10M0 -10L10 0&quot; stroke-linecap=&quot;round&quot; stroke=&quot;#f00&quot; stroke-width=&quot;3&quot;/&gt;&lt;/svg&gt;" result="image"></feImage><feTile in="image" result="tile"></feTile><feComposite operator="in" in="tile" in2="SourceAlpha" result="tiling"></feComposite><feGaussianBlur in="SourceAlpha" stdDeviation="3"></feGaussianBlur><feComponentTransfer><feFuncA type="linear" slope="0.7" intercept="-0.1"></feFuncA></feComponentTransfer><feOffset dy="2" result="shadow"></feOffset><feMerge><feMergeNode in="shadow"></feMergeNode><feMergeNode in="tiling"></feMergeNode><feMergeNode in="specular4"></feMergeNode></feMerge></filter></defs><text x="250" y="75" filter="url(#christmas)">Hello World!</text></svg>