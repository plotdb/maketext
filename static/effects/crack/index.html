<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 500 150"><defs><filter id="crack" x="-100%" y="-100%" width="300%" height="300%"><feFlood flood-color="#000" result="flood"></feFlood><feImage width="1024" height="768" href="/assets/img/texture/soil.jpg" result="image"></feImage><feComposite operator="in" in="image" in2="SourceAlpha" result="comp"></feComposite><feColorMatrix type="matrix" in="comp" result="color1" values="1 0 0 0 0  0 1 0 0 0  0 0 1 0 0  1 0 0 0 0"></feColorMatrix><feColorMatrix type="matrix" in="comp" result="color2" values="0 0 0 0 0  0 0 0 0 0  0 0 0 0 0  -1 -1 -1 1 0"></feColorMatrix><feMerge result="merge"><feMergeNode in="SourceGraphic"></feMergeNode><feMergeNode in="color2"></feMergeNode></feMerge><feGaussianBlur stdDeviation="4" in="SourceAlpha" result="blur"></feGaussianBlur><feSpecularLighting surfaceScale="40" specularConstant="3" specularExponent="3" lighting-color="#ac7c2e" in="blur" result="specular"><fePointLight x="-250" y="-50" z="300"></fePointLight></feSpecularLighting><feComposite operator="in" in="specular" in2="SourceAlpha" result="comp2"></feComposite><feComposite in="merge" in2="comp2" operator="arithmetic" k1="0" k2="1" k3="1" k4="0" result="out"></feComposite><feComposite operator="in" in="out" in2="color1" result="light"></feComposite><feMerge result="merge2"><feMergeNode in="merge"></feMergeNode><feMergeNode in="light"></feMergeNode></feMerge><feTurbulence result="turb" baseFrequency="0.05" numOctaves="10" seed="1"></feTurbulence><feDisplacementMap in="merge2" in2="turb" scale="6"></feDisplacementMap><feOffset dx="-2" dy="-2" result="output"></feOffset><feMerge result="merge3"><feMergeNode in="output"></feMergeNode><feMergeNode in="merge2"></feMergeNode></feMerge><feComposite operator="in" in="flood" in2="merge3" result="merge3-flood"></feComposite><feComponentTransfer><feFuncA type="linear" slope="0.7" intercept="-0.1" result="merge3-comp"></feFuncA></feComponentTransfer><feGaussianBlur in="merge3-comp" stdDeviation="3" result="blur"></feGaussianBlur><feOffset dy="3" in="blur" result="blur-offset"></feOffset><feMerge result="merge4"><feMergeNode in="blur-offset"></feMergeNode><feMergeNode in="merge3"></feMergeNode></feMerge></filter></defs><text x="250" y="75" fill="#340000" filter="url(#crack)">Hello World!</text></svg>