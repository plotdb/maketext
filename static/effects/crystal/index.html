<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 500 150"><defs><filter id="crystal"><feMorphology operator="dilate" radius="2" in="SourceGraphic" result="border"></feMorphology><feFlood flood-color="#000" result="black"></feFlood><feMorphology operator="dilate" radius="2" in="SourceGraphic" result="erode"></feMorphology><feGaussianBlur in="erode" stdDeviation="4" result="blur"></feGaussianBlur><feOffset in="blur" dx="2" dy="2" result="offset"></feOffset><feComposite operator="atop" in="offset" in2="black" result="merge"></feComposite><feComposite operator="in" in="a" in2="SourceGraphic" result="inner-shadow"></feComposite><feGaussianBlur in="SourceGraphic" stdDeviation="2.5" result="shadow1"></feGaussianBlur><feOffset in="shadow1" dx="0" dy="5" result="shadow"></feOffset><feSpecularLighting in="SourceGraphic" result="specular" surfaceScale="1.5" specularConstant="100" specularExponent="10" lighting-color="#fff"><fePointLight x="250" y="-300" z="-400"></fePointLight></feSpecularLighting><feGaussianBlur in="specular" stdDeviation="1" result="specular2"></feGaussianBlur><feMerge result="merge"><feMergeNode in="border"></feMergeNode><feMergeNode in="inner-shadow"></feMergeNode><feMergeNode in="specular2"></feMergeNode></feMerge><feComponentTransfer in="merge" result="final"><feFuncA type="discrete" tableValues="0 0.9"></feFuncA></feComponentTransfer><feMerge><feMergeNode in="shadow"></feMergeNode><feMergeNode in="final"></feMergeNode></feMerge></filter></defs><text x="250" y="75" fill="#ccc" stroke="#666" stroke-width="1.5" filter="url(#crystal)">Hello World</text></svg>