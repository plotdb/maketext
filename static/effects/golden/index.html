<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 500 150"><defs><filter id="golden" x="-100%" y="-100%" width="300%" height="300%"><feFlood flood-color="#806527" result="flood"></feFlood><feFlood flood-color="#fff694" result="flood-light"></feFlood><feComposite operator="in" in="flood" in2="SourceAlpha" result="flood-comp"></feComposite><feGaussianBlur in="flood-comp" stdDeviation="3" result="blur"></feGaussianBlur><feOffset in="blur" dy="3" result="offset-blur"></feOffset><feComponentTransfer in="offset-blur" result="comp-blur"><feFuncA type="linear" slope="0.7" intercept="-0.1"></feFuncA></feComponentTransfer><feSpecularLighting surfaceScale="200" specularConstant="20" specularExponent="1" lighting-color="#fff" in="blur" result="specular"><fePointLight x="-250" y="-250" z="3500"></fePointLight></feSpecularLighting><feComponentTransfer in="specular" result="inspecular"><feFuncA type="linear" slope="-1" intercept="1"></feFuncA></feComponentTransfer><feComposite operator="in" in="flood-light" in2="inspecular" result="inspecular2"></feComposite><feComposite operator="in" in="inspecular2" in2="SourceAlpha" result="light"></feComposite><feMerge><feMergeNode in="comp-blur"></feMergeNode><feMergeNode in="SourceGraphic"></feMergeNode><feMergeNode in="light"></feMergeNode></feMerge></filter></defs><text x="250" y="75" filter="url(#golden)" fill="#f7c61d">Hello World!</text></svg>